# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsWindow.cpp
# Commit: 2b20b560fe61
# Full Hash: 2b20b560fe614c26fe779367c3323addaa55122a
# Author: stransky <stransky@redhat.com>
# Date: 2021-06-20 09:32:48
# Description:
#   Bug 1716796 [Wayland] Poll mozcontainer remap state, r=rmader
#   
#   - Poll mozcontainer remap state by moz_container_wayland_remapped()
#   - Remove WindowSurface::Reset()
#   - Remove moz_container_wayland_set_window_surface() as it adds extra ref at WindowSurface() and causes cyclic dependency.
# ==============================================================================

diff -r 668013f3e201 -r 2b20b560fe61 widget/gtk/nsWindow.cpp
--- a/widget/gtk/nsWindow.cpp	Sat Jun 19 07:06:41 2021 +0300
+++ b/widget/gtk/nsWindow.cpp	Sat Jun 19 09:33:19 2021 +0000
@@ -723,11 +723,9 @@
   // destroys the the gl context attached to it).
   DestroyCompositor();
 
-#ifdef MOZ_X11
   // Ensure any resources assigned to the window get cleaned up first
   // to avoid double-freeing.
   mSurfaceProvider.CleanupResources();
-#endif
 
   ClearCachedResources();
 
